lock/src/main/java/com/auth0/android/lock/views/EnterpriseLayout.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * EnterpriseLayout.java
+ *
+ * Copyright (c) 2016 Auth0 (http://auth0.com)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package com.auth0.android.lock.views;
+
+import android.content.Context;
+import android.support.annotation.IdRes;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.RelativeLayout;
+
+import com.auth0.android.lock.Configuration;
+import com.auth0.android.lock.R;
+import com.squareup.otto.Bus;
+
+public class EnterpriseLayout extends RelativeLayout implements View.OnClickListener {
+    private Bus bus;
+    private Configuration configuration;
+
+    private static final
+    @IdRes
+    int FORM_ID = 8;
+    private static final
+    @IdRes
+    int GO_BACK_BTN_ID = 11;
+
+    public EnterpriseLayout(Context context) {
+        super(context);
+    }
+
+    public EnterpriseLayout(Context context, Bus lockBus, Configuration configuration) {
+        super(context);
+        this.bus = lockBus;
+        this.configuration = configuration;
+        init();
+    }
+
+    private void init() {
+        showDomainForm();
+    }
+
+    @Override
+    public void onClick(View v) {
+        switch (v.getId()) {
+            case GO_BACK_BTN_ID:
+                showDomainForm();
+                break;
+        }
+    }
+
+    private void showDomainForm() {
+        this.removeView(findViewById(FORM_ID));
+
+        DomainFormView domainForm = new DomainFormView(getContext(), this.bus, this.configuration);
+        domainForm.setId(FORM_ID);
```

Comment:
Not sure if this is safe to use, since we are hardcoding the value and not letting android generate it. Also we should probably keep the form reference instead